web/src/views/SubmissionPortal/Components/HarmonizerSidebar.vue

```diff
@@ -0,0 +1,162 @@
+      default: null,
+    },
+    /**
+     * Information about the active template.
+     */
+    harmonizerTemplate: {
+      type: Object as PropType<HarmonizerTemplateInfo>,
+      required: true,
+    },
+    /**
+     * Whether the current user is allowed to edit metadata.
+     */
+    metadataEditingAllowed: {
+      type: Boolean,
+      default: false,
+    },
+    /**
+     * The Harmonizer API instance.
+     */
+    harmonizerApi: {
+      type: Object as PropType<HarmonizerApi>,
+      required: true,
+    },
+  },
+  emits: ['export-xlsx', 'import-xlsx'],
+  setup(props, { emit }) {
+    const tabModel = ref(0);
+    const TABS = [
+      {
+        icon: 'mdi-information-outline',
+        label: 'Column Help',
+      },
+      {
+        icon: 'mdi-text-search',
+        label: 'Find & Replace',
+      },
+      {
+        icon: 'mdi-assistant',
+        label: 'Metadata Suggester',
+      },
+      {
+        icon: 'mdi-swap-vertical',
+        label: 'Import & Export',
+      },
+      {
+        icon: 'mdi-help-circle-outline',
+        label: 'Help',
+      },
+    ];
+
+    const handleImport = (...args: never[]) => {
+      emit('import-xlsx', ...args);
+    };
+
+    return {
+      TABS,
+      handleImport,
+      tabModel,
+    };
+  },
+});
+</script>
+
+<template>
+  <div class="harmonizer-sidebar-content">
+    <v-tabs
+      v-model="tabModel"
+      grow
+    >
+      <v-tooltip
+        v-for="tab in TABS"
+        :key="tab.label"
+        open-delay="600"
+        top
+        z-index="400"
+      >
+        <template #activator="{ on, attrs }">
+          <v-tab
+            v-bind="attrs"
+            v-on="on"
+          >
+            <v-icon>{{ tab.icon }}</v-icon>
+          </v-tab>
+        </template>
+        <span>{{ tab.label }}</span>
+      </v-tooltip>
+    </v-tabs>
+
+    <v-divider />
+
+    <v-tabs-items v-model="tabModel">
+      <v-tab-item>
+        <ColumnHelp
+          :column-help="columnHelp"
+          :harmonizer-template="harmonizerTemplate"
+          @full-reference-click="harmonizerApi.launchReference()"
+        />
+      </v-tab-item>
+      <v-tab-item>
+        <FindReplace
+          :harmonizer-api="harmonizerApi"
+        />
+      </v-tab-item>
+      <v-tab-item>
+        <MetadataSuggester
+          :enabled="metadataEditingAllowed"
+          :harmonizer-api="harmonizerApi"
+          :schema-class-name="harmonizerTemplate.schemaClass"
+        />
+      </v-tab-item>
+      <v-tab-item>
+        <ImportExportButtons
+          :import-disabled="!metadataEditingAllowed"
+          @export="$emit('export-xlsx')"
+          @import="handleImport"
+        />
+      </v-tab-item>
+      <v-tab-item>
+        <ContactCard elevation="0" />
+      </v-tab-item>
+    </v-tabs-items>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.harmonizer-sidebar-content {
+  font-size: 14px;
+
+  .v-tab {
+    min-width: 10px;
+  }
+}
+</style>
```

Comment:
What about adding a border and/or some height/shadow to the sidebar to make it feel more separate from the data harmonizer?

Reply:
I did this via adding a few `v-navigation-drawer` props:

* `temporary` -- this adds a box shadow to the drawer element
* `hide-overlay` -- we don't want to dim the rest of the page content; we just want the shadow
* `stateless` -- probably should have been using this previously since we fully manage the open/closed state and we don't need mobile support